client/src/components/NotFound/index.tsx

```diff
@@ -1,7 +1,23 @@
+import { Result, Button } from 'antd';
 import * as React from 'react';
+import { useHistory } from 'react-router';
+import { URL_HOME } from '../../constants';
 
 const NotFound: React.FC = () => {
-  return <div>404!</div>;
+  const history = useHistory();
+
+  return (
+    <Result
+      status="404"
+      title="404"
+      subTitle="Sorry, the page you are trying to visit does not exist."
+      extra={
+        <Button key="home" onClick={() => history.push(URL_HOME)} type="primary">
+          Back Home
```

Comment:
Seems like we'll have to wait for #41 to be merged and update this component accordingly to support multiple languages